src/test/java/io/arlas/server/wfs/WFSServiceIT.java

```diff
@@ -0,0 +1,14 @@
+package io.arlas.server.wfs;
+
+import org.junit.Test;
+//import org.opengis.cite.iso19142.TestNGController;
+
+public class WFSServiceIT {
```

Comment:
Is this test working for real? It looks like nothing is executed...

Reply:
Deleted